### DISCUSSION TITLE
Default combo colors in the new editor setup screen

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#30122] [Q&A] [answered]
In the new editor setup screen these 4 combo colors are shown by default:
![zzzzzz](https://github.com/user-attachments/assets/0c044708-a5e2-4da9-b04f-452d9cae839a)

But this is not correct, because the default behavior of a new map is indeed to follow skin settings. Take argon skin as an example:
![zzzzzzz2](https://github.com/user-attachments/assets/5d4a3a0c-1281-4067-b153-3ef80f081508)

Compare stable, where skin combo colors are displayed when the "enable custom colors" is not checked (stable doesn't have argon so used another skin):
![z3](https://github.com/user-attachments/assets/17ca4ce2-5b51-45bc-b682-1e0f3ada85b1)

I'm not exactly sure what's the best behavior of this. Basically there are two questions:

1: should new maps have custom combo colors set by default? I think it's required by the RC to set custom colors ("Each beatmap must use at least two different custom combo colours unless the default skin is forced"), so maybe it's a desired behavior, though being a departure from stable.

2a: if the answer to 1 is yes, then what default colors should be used? I think it should follow the active skin when the map is created, but that's only my own opinion.

2b: if the answer to 1 is no, then what should be displayed in the "colors" region by default? Maybe no colors at all (as the old setup screen), or the skin colors (as in stable), along with the "enable custom colors" checkbox.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/pull/30108#issuecomment-2393242265
